lib/cli/src/common.rs

```diff
@@ -13,9 +13,9 @@ pub struct WasmFeatures {
     #[clap(long = "enable-simd")]
     pub simd: bool,
 
-    /// Enable support for the threads proposal.
-    #[clap(long = "enable-threads")]
-    pub threads: bool,
```

Comment:
We probably don't want to break CLI users that use this flag in scripts etc.

We could leave an `enable-threads` flag that is a no-op, with a comment saying it's deprecated, doesn't do anything,  and mention  ---disable-threads.

Reply:
ah yeah, true